### DISCUSSION TITLE
Advertising ID Declaration Form?

### DISCUSSION THREAD

**Discussion by ponarangan** [airsdk/Adobe-Runtime-Support#2032] [Q&A]
I have been asked to fill advertising ID declaration form by google publish  Does Third party ANE include"  com.google.android.gms.permission.AD_ID " by default or do i have to include it?

**Comment by marchbold** [comment]
`com.google.android.gms.permission.AD_ID` is a permission that you need to define in your manifest, so unless you are using apm to generate your manifest it will only be in your manifest if you include it. 

ANE's have no facility to add permissions like this.

**Reply by ponarangan** [reply]
We are Using the manifest given in the animate. Not in the apm. So what should in the declaration form. Should we give Yes or No.

**Reply by marchbold** [reply]
This is not a yes or no, but either you include the permission or you don't. Whether you do or not is up to you and what privacy policy etc you are adhering too
